app/i18n/en/gen.php

```diff
@@ -2,7 +2,7 @@
 
 return array(
 	'action' => array(
-		'actualize' => 'Actualize',
+		'actualize' => 'Updating',
```

Comment:
I think this one is wrong:
![image](https://cloud.githubusercontent.com/assets/1008324/24577685/c8e653c6-16d2-11e7-8e7d-1a99113994ce.png)
Should be something like *Actualize* or *Update*